README.md

```diff
@@ -235,18 +235,22 @@ jq -r '.name + "," + .role + "," + (.default_group_id|tostring) + "," + (.active
 sh scripts/merge-agent-and-group-description.sh
 ```
 
-### Adhoc Scripts
+### Other Scripts
 
-#### De-duplicate support queue - manual script
-The purpose of this script is to identify multiple tickets from the same UserID which match a set of parameters. If there are > 1 not closed or solved, they are moved to a dedicated Zendesk queue.
+#### De-duplicate support queue
+* The script uses the Zendesk API to identify multiple tickets from the same UserID in the 1st line support queue which match a set of parameters. If there are > 1 tickets with a status of not closed or solved, they are moved to a dedicated Zendesk queue.
+* The script is automated via a Concourse pipeline called 'zendesk-deduplicator'. On a successful merge to master, a new docker container is built, tagged and stored. Following the schedule, the latest version of the container is retrieved and the ruby script executed. Secrets are stored using AWS SSM.
+* The script requires sufficient permissions in Zendesk and is configured to use account email and a token as auth.
 
-Edit window required (e.g. 12 hours) in this line, e.g.:
+To use the script from the CLI, set the environment variables as above, then:
```

Comment:
```suggestion
To use the script from the CLI, set the environment variables as [above](#add-environmental-variables-to-bashrc-or-to-screen-session), then:
```

Can we add the link to allow the user to find the relevant section about adding environment variables quickly?